package.json

```diff
@@ -28,6 +28,7 @@
     "fastify": "~4.13.0",
     "fastify-cli": "^5.7.1",
     "fastify-plugin": "~4.5.0",
+    "graphql": "^16.7.1",
```

Comment:
Is graphql used?

Reply:
This was added due to a peer dependency issue earlier - so I think one of the packages do without specifying it.